cypress/e2e/createVault.cy.ts

```diff
@@ -34,7 +34,7 @@ context('Coordinape', () => {
     cy.login();
     cy.wait(1000);
     cy.contains('Ended Epoch With Gifts').click();
-    cy.contains('Vaults').click();
+    cy.contains('Vaults').click({ force: true });
```

Comment:
I tried removing these and running cypress, and it passed for me, so if possible I'd keep this off unless there's a good reason we need to use it or want to have the cypress runs look less like a users behavior.

Reply:
@crabsinger Yes, I forgot to remove them for this branch.  But I think they will be needed in the other patch branch if we have to move to the `display:none` method to prevent flickering